pkg/codegen/docs/gen.go

```diff
@@ -182,19 +182,12 @@ type enum struct {
 	DeprecationMessage string
 }
 
-// apiTypeDocLinks represents the links for a type's input and output API doc.
-type apiTypeDocLinks struct {
-	InputType  string
-	OutputType string
-}
-
 // docNestedType represents a complex type.
 type docNestedType struct {
-	Name        string
-	AnchorID    string
-	APIDocLinks map[string]apiTypeDocLinks
```

Comment:
This is the map that used to contain the `input` and `output` links for each object type, aka Supporting Types.